src/modules/image/imageApi.ts

```diff
@@ -40,6 +40,13 @@ export const updateImage = (imageMetadata: UpdatedImageMetadata): Promise<ImageA
     body: JSON.stringify(imageMetadata),
   }).then(resolveJsonOrRejectWithError);
 
+export const patchImage = (id: number, formData: FormData): Promise<ImageApiType> =>
+  fetchAuthorized(`${baseUrl}/${id}`, {
+    method: 'PATCH',
+    headers: { 'Content-Type': undefined },
```

Comment:
`headers: { 'Content-Type': undefined },` undefined? what is the reasoning?

Reply:
Min reasoning var det her er sånn det ser ut på POST endepunktet så derfor gjør jeg det samme.
Tror tanken er at vi defaulter til `application/json` men formData'en vi submitter er både json og en binærfil så det blir feil.
Mulig det finnes noe bedre, men dunno :small_airplane:

Reply:
Multipart eller nokke sånt?

Reply:
Ser riktig ut med `multipart/form-data` :shrug: